.gitattributes

```diff
@@ -1,2 +1,2 @@
 **/vendor/** linguist-vendored
-kernel/*.x linguist-language=text
+kernel/config-*.x linguist-language=text
```

Comment:
ITYM `kernel/config-*.x-*` due to the arch suffix, you are trying to match e.g. `kernel/config-4.9.x-x86_64`. 

Actually `kernel/config-*` would work better since it would catch `kernel/config-dbg` too.

Reply:
Ah, forgot about the arch suffix. pushed an update